docs/HowToFAQ/FAQForall.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How do I use forall statements and expressions in a lemma?
+---
+
+## Question
+
+If I'm trying to prove a lemma in Dafny with a forall statement that needs help in the body ( {} ) of the lemma, how do I make an arbitrary variable (specifically of type string32 ) in the body of the lemma to help prove the forall statement?
+
+## Answer
+
+Note that there is a difference between a forall expression, which is a universally quantified formula, and a forall statement, which is used to establish the truth of a universally quantified formula. To do what you want to do, write:
```

Comment:
Maybe make `forall` verbatim throughout here?

Reply:
And throughout the rest of the entries, it might be nice to more consistently make keywords verbatim, though I wouldn't consider it super high priority.

Reply:
OK. I do when I remember and will look for those elsewhere.